### PR TITLE
[RSC] Changelog - Add 'permissionGrants' navigation property to 'chat'

### PR DESCRIPTION
**Pull request by jessiechan27**

### CODE REVIEW

File: api-reference/beta/resources/chat.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ not all scenarios are possible. It is possible to get chat IDs with delegated pe
 | createdDateTime| dateTimeOffset|  Date and time at which the chat was created. Read-only.|
 | lastUpdatedDateTime| dateTimeOffset|  Date and time at which the chat was renamed or list of members were last changed. Read-only.|
 | chatType| [chatType](../resources/chat.md#chattype-values) | Specifies the type of chat. Possible values are:`group`, `oneOnOne` and `meeting`.|
+| permissionGrants| resourceSpecificPermissionGrant| Permissions granted to the apps in the chat.|
```
Comment (subray2014) [inline api-reference/beta/resources/chat.md:59]: Should be a link to resourcePermissionGrant page if such a thing exists.

Comment (AkJo) [inline api-reference/beta/resources/chat.md:59]: [resourceSpecificPermissionGrant] (resourceSpecificPermissionGrant.md) collection

If resourceSpecificPermissionGrant.md does not exist, then please add it.

Comment (AkJo) [inline api-reference/beta/resources/chat.md:59]: We also need a (../api/chat-list-permissionGrants.md) that calls out how the API is invoked and what permissions are needed along with examples.

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 2]
```diff
@@ -73,6 +73,24 @@
       "CreatedDateTime": "2021-03-04T01:13:01.454Z",
       "WorkloadArea": "Teamwork",
       "SubArea": ""
+    },
+    {
```
Comment (AkJo) [inline changelog/Microsoft.Teams.GraphSvc.json:77]: Add an a change log entry for the API page (and resource page if necessary).

**Review by AkJo — APPROVED** [review]
Please apply comments and merge

**Pull request closed without merging**